test/SigningKeys.t.sol

```diff
@@ -0,0 +1,351 @@
+// SPDX-FileCopyrightText: 2023 Lido <info@lido.fi>
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity 0.8.24;
+
+import "forge-std/Test.sol";
+import "forge-std/console.sol";
+
+import { SigningKeys } from "../src/lib/SigningKeys.sol";
+import { Utilities } from "./helpers/Utilities.sol";
+
+// Wrap the library internal methods to make an actual call to them.
+// Supposed to be used with `expectRevert` cheatcode and to pass
+// calldata arguments.
+contract Library {
+    function saveKeysSigs(
+        uint256 nodeOperatorId,
+        uint256 startIndex,
+        uint256 keysCount,
+        bytes memory pubkeys,
+        bytes memory signatures
+    ) external returns (uint256) {
+        return
+            SigningKeys.saveKeysSigs(
+                nodeOperatorId,
+                startIndex,
+                keysCount,
+                pubkeys,
+                signatures
+            );
+    }
+
+    function removeKeysSigs(
+        uint256 nodeOperatorId,
+        uint256 startIndex,
+        uint256 keysCount,
+        uint256 totalKeysCount
+    ) external returns (uint256) {
+        return
+            SigningKeys.removeKeysSigs(
+                nodeOperatorId,
+                startIndex,
+                keysCount,
+                totalKeysCount
+            );
+    }
+
+    function loadKeys(
+        uint256 nodeOperatorId,
+        uint256 startIndex,
+        uint256 keysCount
+    ) external view returns (bytes memory pubkeys) {
+        return SigningKeys.loadKeys(nodeOperatorId, startIndex, keysCount);
+    }
+
+    function loadKeysSigs(
+        uint256 nodeOperatorId,
+        uint256 startIndex,
+        uint256 keysCount,
+        bytes memory pubkeys,
+        bytes memory signatures,
+        uint256 bufOffset
+    ) external view {
+        SigningKeys.loadKeysSigs(
+            nodeOperatorId,
+            startIndex,
+            keysCount,
+            pubkeys,
+            signatures,
+            bufOffset
+        );
+    }
+}
+
+contract SigningKeysTest is Test, Utilities {
+    Library signingKeys;
+
+    function setUp() public {
+        signingKeys = new Library();
+    }
+
+    function testFuzz_saveKeysSigs(
+        uint256 nodeOperatorId,
+        uint256 startIndex,
+        uint256 keysCount
+    ) public {
+        vm.assume(keysCount > 0);
+        vm.assume(keysCount < 100);
+        unchecked {
+            vm.assume(startIndex + keysCount > startIndex);
```

Comment:
it looks like `keysCount > 0`, which is above already

Reply:
This is an overflow check for sum